erpnext/hr/doctype/appraisal/appraisal.json

```diff
@@ -1,775 +1,254 @@
 {
- "allow_copy": 0, 
- "allow_events_in_timeline": 0, 
- "allow_guest_to_view": 0, 
- "allow_import": 0, 
- "allow_rename": 0, 
- "autoname": "naming_series:", 
- "beta": 0, 
- "creation": "2013-01-10 16:34:12", 
- "custom": 0, 
- "docstatus": 0, 
- "doctype": "DocType", 
- "document_type": "Setup", 
- "editable_grid": 0, 
+ "actions": [],
+ "autoname": "naming_series:",
+ "creation": "2013-01-10 16:34:12",
+ "doctype": "DocType",
+ "document_type": "Setup",
+ "engine": "InnoDB",
+ "field_order": [
+  "employee_details",
+  "naming_series",
+  "kra_template",
+  "employee",
+  "employee_name",
+  "column_break0",
+  "status",
+  "start_date",
+  "end_date",
+  "department",
+  "section_break0",
+  "goals",
+  "total_score",
+  "section_break1",
+  "remarks",
+  "other_details",
+  "company",
```

Comment:
The value of the company should be fetched from employee master. Keep the field read-only doesn't allow to change.
![image](https://user-images.githubusercontent.com/32095923/95314468-f6084180-08ae-11eb-93f5-2609304c7900.png)